starter-code/src/functions-and-arrays.js

```diff
@@ -74,6 +165,23 @@ var wordsCount = [
   'disobedience',
   'matter'
 ];
+
+function howManyTimes(wordsCount) {
```

Comment:
You would probably need to pass 2 parameters into this function in order to have a word to check how many times the word is in the array you pass in